### ISSUE TITLE
make install in README

### ISSUE THREAD

**Issue by jmgate** [alan-turing-institute/the-turing-way-book#3]
Trying to build this on my own, I was following these instructions in the [README](../blob/master/README.md):

### Locally (Mac / Linux Only)
To install jupyter-book etc.
```bash
cd website
pip install -r requirements.txt
```

If you want to see your local changes on your own computer you'll have to go in the website directory and enter make serve (type make on it's own to see the other options).
```bash
cd website
make site && make serve
```

You might want to note that the first time you do this, you should probably do a `make install` in the `website` directory&mdash;at least I had to.  If not, feel free to close.

**Comment by alexmorley** [comment]
Yep good spot. In fact we should probably add the pip install step to the makefile so that it's just one command.

**Comment by alexmorley** [comment]
To do:
- [ ] Add `pip install -r requirements.txt` to `install:` section of website/Makefile
- [ ] Change that section of the docs to `cd website; make install`
